app/controllers/request-to-go-live/agreement/get.controller.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+// NPM dependencies
+const lodash = require('lodash')
+
+// Local dependencies
+const goLiveStage = require('../../../models/go-live-stage')
+const { requestToGoLive } = require('../../../paths')
+const response = require('../../../utils/response')
+
+module.exports = (req, res) => {
+  if (req.service.currentGoLiveStage === goLiveStage.CHOSEN_PSP_STRIPE ||
```

Comment:
A useful pattern for checking if a value is in a certain range is 
```js
// false
[ 'many', 'options' ].includes('invalid')
// true
[ 'many', 'options' ].includes('options')
```

In this case it could read: 
```js
const { CHOSEN_PSP_STRIPE, CHOSEN_PSP_WORLDPAY, CHOSEN_PAP_SMARTPAY, CHOSEN_PSP_EPDQ } = goLiveStage 
const chosenOptions = [ CHOSEN_PSP_STRIPE, CHOSEN_PSP_WORLDPAY, CHOSE_PSP_SMARTPAY, CHOSEN_PSP_EDPQ ]

const requestInChosenStage = chosenOptions.includes(req.service.currentGoLiveStage) 

if (requestInChosenStage) { 
  // ...
}
```

Reply:
Thanks! 👍